vktrace/vktrace_common/vktrace_packet_id_defs.h

```diff
@@ -0,0 +1,195 @@
+/*
+ *
+ * Copyright (C) 2015-2017 Valve Corporation
+ * Copyright (C) 2015-2017 LunarG, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Mark Lobodzinski <mark@lunarg.com>
+ */
+
+#pragma once
+
+// Note: Changing the following enums will break backwards-compatibility with
+//       previously generated traces.  New values should be appended, deprecated
+//       values should be left in or skipped.
+enum VKTRACE_TRACE_PACKET_ID_VK {
+    VKTRACE_TPI_VK_vkApiVersion = 7,

```

Comment:
I understand the reason we start at 7 is the fact that this enum table continuing from VKTRACE_TRACE_PACKET_ID. in this first enum currently VKTRACE_BEGIN_API_HERE is 7 but this can change if new enums added to first enum table. Thus making this list start at 7 is still risky. I think we can merge VKTRACE_TRACE_PACKET_ID enum into this table (as this is not generated anymore) and keep them all in same palce